week-6/Homework/mandatory/2-quotegenerator/quotes.js

```diff
@@ -489,3 +490,19 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+
+function qoutesGenerator(){
```

Comment:
Nice, you were close! Just two things to fix to make it work:
- move all the code from the function quotesGenerator outside (there is no need to wrap it in a function)
- move the script tag in your html below the body (the problem is that, at the moment, you are running the JS before all the HTML has been loaded, so the JS doesn't find the HTML elements